src/linear_algebra/finite_dimensional.lean

```diff
@@ -88,9 +88,22 @@ open is_noetherian
 
 section division_ring
 
-variables (K : Type u) (V : Type v) [division_ring K] [add_comm_group V] [module K V]
+variables {K : Type u} {V : Type v} [division_ring K] [add_comm_group V] [module K V]
 {V₂ : Type v'} [add_comm_group V₂] [module K V₂]
 
+/-- If the codomain of an injective linear map is finite dimensional, the domain must be as well. -/
+def of_injective (f : V →ₗ[K] V₂) (w : function.injective f)
+  [finite_dimensional K V₂] : finite_dimensional K V :=
+begin
+  haveI : is_noetherian K V₂ := is_noetherian.iff_fg.mpr ‹_›,
+  exact module.finite.of_injective f w,
+end
```

Comment:
```suggestion
let i : is_noetherian K V₂ := is_noetherian.iff_fg.mpr ‹_› in
  by exactI module.finite.of_injective f w
```
presumably works.

Reply:
I guess? I'm not sure that this is any improvement, and the style guide recommends against using `by` unless it fits on a single line.

Reply:
Oh, I see it does fit on one line, it was just github reflowing.